### DISCUSSION TITLE
OSError: .local/lib/python3.8/site-packages/nvidia/cublas/lib/libcublas.so.11: undefined symbol: cublasLtGetStatusString, version libcublasLt.so.11

### DISCUSSION THREAD

**Discussion by tassosblackg** [explosion/spaCy#12236] [Help: Installation] [answered]
## How to reproduce the behaviour

import spacy

## Your Environment

* Operating System: Ubuntu 20.4 LTS
* Python Version Used: 3.8.10
* spaCy Version Used: 3.5.0
*  nvcc -version : cuda_11.3.r11.3

What is wrong something with installation is going wrong, actually it happens after installing the `en_core_web_trf` , some of this packages `torch-1.13`, `transformers-4.26.0`, or `spacy-transformers-1.2.1` is causing this problem, maybe it is the torch installation

I manage to follow again the installation process on another PC and everything is running smooth.

**Comment by rmitsch** [comment, marked as answer]
Hi @tassosblackg, it's hard to diagnose this issue with the available information. We suspect that this might be related to a change in PyTorch from v1.13 upwards - CUDA libraries are now dynamically linked (instead of statically in < 1.13). See e. g. [here](https://stackoverflow.com/questions/74394695/how-does-one-fix-when-torch-cant-find-cuda-error-version-libcublaslt-so-11-no)) for possible workarounds.

**Reply by tassosblackg** [reply]
Yeah I've seen that, I followed it but is not a completely right approach. I managed to solve the issue by installing `pytorch =1.13 +cpu` before installing the `en_core_web_trf`, in order to avoid by default installation of `pytorch` that leads to the above issue. 

So, I've install `spacy` then `pytorch` and then downloaded the `en_core_web_trf`. Seems to work at the moment.
